src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -63,6 +63,9 @@
     text: |
         end=`date -v+30M '+%Y-%m-%dT%H:%MZ'`
         az storage account generate-sas --permissions cdlruwap --account-name MyStorageAccount --services qt --resource-types sco --expiry $end -otsv
+  - name: Generates a shared access signature for the account (autogenerated)
+    text: az storage account generate-sas --account-key 00000000 --account-name $my_diagnostic_storage_account --expiry $end -otsv --https-only --permissions acuw --resource-types co --services bfqt
```

Comment:
`$my_diagnostic_storage_account` is bash-specific syntax. Why not simply use the normal "myStorageAccount" name convention?